backend/utils/pypi.py

```diff
@@ -72,13 +72,24 @@ def format_plugin(plugin: dict) -> dict:
     """
     version = get_attribute(plugin, ["info", "version"])
 
+    # parse raw author names string 
+    raw_name = get_attribute(plugin, ["info", "author"])
+    # currently splitting by "&", ",", and the word "and"
+    regexp = r'&|,|\sand\s'
+    author_names = re.split(regexp, raw_name)
+    author_names = [name.strip() for name in author_names if name is not None]
+    authors = []
+    for name in author_names:
+        if name:
+            authors.append({'name': name, 'email': get_attribute(plugin, ["info", "author_email"])})
```

Comment:
Consider using a list comprehension and reducing the number of calls to `get_attribute()`.  I find this more clear:
```suggestion
    # UNTESTED SNIPPET
    author_email=get_attribute(plugin, ["info", "author_email"])
    authors = [
        dict(name=name, email=author_email)
        for name in author_names
        if name
    ]

```

Reply:
Oh good idea! I think we're actually depreciating the 'email' entry, so I'm going to remove that, but I'll switch to using list comprehension.

Reply:
I <3 list comprehensions